README.md

```diff
@@ -1,2 +1,29 @@
 # ecephys_etl_pipelines
 Pipelines and modules for processing extracellular electrophysiology data 
+Align Timestamps
+================
+Computes a transformation from probe sample indices to times on the experiment master clock, then maps zero or more timestamp 
+arrays through that transform.
+
+
+Running
+-------
+```
+python -m allensdk.brain_observatory.ecephys.align_timestamps --input_json <path to input json> --output_json <path to output json>
+```
+See the schema file for detailed information about input json contents.
+
+
+Input data
+----------
+- Sync h5 : Contains information about barcode pulses assessed on the master clock
+- For each probe
+    - barcode channel states file: lists rising and falling edges on the probe's barcode line
+    - barcode timestamps file: lists probe samples at which rising and falling edges were detected
+    - mappable timestamp files: Will be transformed to the master clock. An example would be a file listing timestamps of detected spikes.
+
+
+
+Output data
+-----------
+Each mappable file for each probe is aligned and written out. Additionally, the transform is written into the output json. 
```

Comment:
i know just a copy, and there is a schema, but, this isn't much information here.